BlogTemplate/Pages/New.cshtml

```diff
@@ -1,27 +1,37 @@
 ﻿@page
 @using BlogTemplate
 @using BlogTemplate.Models
-@*@model Models.Blog*@
 @inject Blog Blog
 @model NewModel
 @using Microsoft.AspNetCore.Mvc.RazorPages
 @addTagHelper "*, Microsoft.AspNetCore.Mvc.TagHelpers"
 
 @{
     ViewData["Title"] = "New Blog Post";
+    string tagsString;
+    if (Model.Post != null)
+    {
+        tagsString = string.Join(", ", Model.Post.Tags);
+    }
+    else
+    {
+        tagsString = string.Empty;
+    }
 }
 
 <h2>Write a New Post</h2>
 <form method="POST">
-    <p><h3><label for="Title">Title</></label></h3></p>
-    <p><input type="text" name="Title" asp-for="Post.Title" /></p>
+    <div asp-validation-summary="All"></div>
+    <p><h4><label for="Title">Title</></label></h4></p>
```

Comment:
We shouldn't be using `<h4>` on these labels.  (a) `<h4>` doesn't belong inside a `<p>`, they are different semantically.  (b) if you're using it to get a visual effect, we should do that through a CSS class instead.

Similarly, most of these `<p>` tags should go away and just use `<label>` and `<input>`.